app/models/career.rb

```diff
@@ -1,5 +1,11 @@
 class Career < ApplicationRecord
-  before_create :set_free_ranks
+  before_validation :set_free_ranks, on: :create
```

Comment:
now that there are not validations, should we change this to a `before_create` callback?